storage/data-plane/src/test/java/com/microsoft/azure/storage/TransferManagerTest.groovy

```diff
@@ -655,8 +583,11 @@ class TransferManagerTest extends APISpec {
     @Unroll
     def "Download file IA null"() {
         when:
-        TransferManager.downloadBlobToFile(file, blobURL, null, null).blockingGet()
-
+        if(file != null) {
+            TransferManager.downloadBlobToFile(AsynchronousFileChannel.open(file.toPath()), blobURL, null, null).blockingGet()
```

Comment:
What was the reason for this change?

Reply:
After upgrading netty.
The File cannot be casted to AsynchronousFileChannel casting issue came up. downloadBlobToFile expects AsynchronousFileChannel.
so created the AsynchronousFileChannel creation from file.